### ISSUE TITLE
Option to have graphs in a single column or all graphs with the same width

### ISSUE THREAD

**Issue by rogerchi** [cloudashdev/cloudash#80]
Would make it easier to compare across different graphs if the x-axis lined up.

**Comment by mthenw** [comment]
Thanks for reporting. Just so I understand, do you mean graphs on the Overview page?

**Comment by rogerchi** [comment]
Yes, the Overview page.  Maybe for wide enough monitors there could be two separate columns for API and Lambda
